### DISCUSSION TITLE
Analog Inputs

### DISCUSSION THREAD

**Discussion by MARIOBASZ** [phil-barrett/grblHAL-teensy-4.x#67] [Ideas]
Hi Phill, great job, I'm interested in buying one of your plates, the drawback is that Tindie doesn't ship to Argentina (I'll try to find a solution). I will need analog inputs (minimum one, ideal 3).
Can some of the pins be used as analog input?
Greetings Mario.

**Comment by phil-barrett** [comment]
There are no analog inputs on the board.
